flink-streaming-java/src/main/java/org/apache/flink/streaming/api/windowing/windows/TimeWindow.java

```diff
@@ -236,6 +236,6 @@ public int compare(TimeWindow o1, TimeWindow o2) {
 	 * @return window start
 	 */
 	public static long getWindowStartWithOffset(long timestamp, long offset, long windowSize) {
```

Comment:
Is the windowSize actually slide size? I see that it is called by passing the slide instead of size `TimeWindow.getWindowStartWithOffset(timestamp, offset, slide);`